### PR TITLE
Fixed the total deletion caused when a deleted resource was null

### PR DESCRIPTION
**Pull request by GQH1993**
My modifications are on lines 714-716.
Fixed the total deletion caused when a deleted resource was null.

### CODE REVIEW

**Comment by GQH1993** [comment]
> 一位管理员可以验证此补丁吗？

I think so.

**Comment by rohanKanojia** [comment]
ok to test

File: kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java [hunk 1]
```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright (C) 2015 Red Hat, Inc.
- *
+ * <p>
```
Comment (rohanKanojia) [inline kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java:3]: Could you please revert to old license header? Maybe it was caused by auto format?

File: kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java [hunk 2]
```diff
@@ -709,6 +711,9 @@ public Boolean delete(List<T> items) {
     boolean deleted = true;
     if (items != null) {
       for (T item : items) {
+        if (null == item) {
```
Comment (rohanKanojia) [inline kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java:714]:
```suggestion
        if (item == null) {
```

File: kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java (hunk 1 above)
Comment (GQH1993) [reply kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java:3]: you're right

File: kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java (hunk 2 above)
Comment (GQH1993) [reply kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java]: I fixed the format problem

**Comment by rohanKanojia** [comment]
These seems to be some problem with our integration tests on Kubernetes. They are not related to your change

**Comment by manusa** [comment]
Sorry, messed up your origin/master branch when rebasing your changes. I reopened your PR in #2417 since I cannot fix the problem (GQH1993:master became protected after the mess up)

**Comment by GQH1993** [comment]
> Sorry, messed up your origin/master branch when rebasing your changes. I reopened your PR in #2417 since I cannot fix the problem (GQH1993:master became protected after the mess up)

Alright. Thank you.

**Pull request closed without merging**